runelite-client/src/main/java/net/runelite/client/ui/components/IconTextField.java

```diff
@@ -161,6 +159,12 @@ public void setEditable(boolean editable)
 		}
 	}
 
+	@Override
+	public boolean requestFocusInWindow()
+	{
+		return textField.requestFocusInWindow(); 
```

Comment:
You should call super.requestFocusInWindow() here too.

Reply:
Why?

Reply:
Because swing. I dont want to take bets

Reply:
Was about to ask, the parent is a jpanel, and it works as intended. But if you feel it's better to have it, sure.